### ISSUE TITLE
After login go back to page where login was initiated

### ISSUE THREAD

**Issue by acka47** [hbz/oerworldmap#789]
Currently, I find myself on the home page after login no matter on which page I clicked on the login button. Expected behaviour is to land on the page where I was when clicking the "login" button.

**Comment by karindr** [comment]
I agree. It would be better to land on the page where I was when clicking the "login" button.

**Comment by literarymachine** [comment]
On beta

**Comment by karindr** [comment]
I logged in when I was on a resource and got there from a list. After I logged in I was shown only the list.
Start:
https://beta.oerworldmap.org/resource/?q=&filter.about.%40type=Service#urn:uuid:88bdfe83-9dbd-471f-ab89-1f5989c7d026
After login:
https://beta.oerworldmap.org/resource/?q=&filter.about.%40type=Service

**Comment by acka47** [comment]
Same here. But I think this is already better than before and the `#` part can't be taken into account by the server anyway. Thus, +1.

**Comment by literarymachine** [comment]
> I logged in when I was on a resource and got there from a list. After I logged in I was shown only the list.

Fixed on beta.

**Comment by karindr** [comment]
+1

**Comment by karindr** [comment]
There is a new phenomenon here:
When I do a search and end up on an entry page (not zoomed in) and then I log in I stay on the entry view but the map is now zoomed in. I expect the map not to change when logging in.

**Comment by acka47** [comment]
This does not work on the new map at all.
